server/integration/feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/search/engine/main/module.xml

```diff
@@ -8,6 +8,7 @@
         <module name="javax.transaction.api" />
         <module name="org.hibernate.commons-annotations" />
         <module name="org.apache.lucene" export="true" />
+        <module name="org.apache.lucene.internal" export="true" />
```

Comment:
This sounds a bit strange, an "internal" that is exported? Maybe we should move whatever need to be exported to the "public" module?